src/utils/index.js

```diff
@@ -13,3 +13,13 @@ export const queryAll = (selector, context = document) => {
 
 	return Array.prototype.slice.call(context.querySelectorAll(selector));
 };
+
+const escapeCssRegex = /(^[^_a-zA-Z\u00a0-\uffff]|[^-_a-zA-Z0-9\u00a0-\uffff])/g;
+
+export const escapeCssIdentifier = (ident) => {
+	if (window.CSS && window.CSS.escape) {
+		return CSS.escape(ident);
+	} else {
+		return ident.replace(escapeCssRegex, "\\$1");
+	}
+};
```

Comment:
The CSS escape seems to be very much supported. Do we need the fallback at all?

Reply:
Happy to leave it out. Since global support is at 93% of devices, we should probably still leave it in a conditional. Worst case, nothing is escaped in pre-Chrome Edge.